core/api/src/domain/authentication/index.ts

```diff
@@ -35,6 +35,20 @@ export const getSupportedCountries = ({
   return countries
 }
 
+export const getNextPageToken = (link: string): string | undefined => {
```

Comment:
this is specific to service implementation, should not be in the core domain

Reply:
https://github.com/GaloyMoney/galoy/pull/3943/commits/ca20f97e8d59905ea4e665ec9111fbedfc3a6fed